build-system/tasks/runtime-test/status-report.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const argv = require('minimist')(process.argv.slice(2));
+const log = require('fancy-log');
+const request = require('request');
+const {cyan, yellow} = require('ansi-colors');
+const {gitCommitHash} = require('../../git');
+const {isTravisPullRequestBuild} = require('../../travis');
+
+const reportBaseUrl = 'https://amp-test-status-bot.appspot.com/v0/tests/';
+
+function inferTestType() {
+  if (argv['local-changes']) {
+    return 'local-changes';
+  }
+  if (argv.single_pass) {
+    return null;
```

Comment:
Any reason single pass is being skipped? @erwinmombay would it be useful to have these results show for PRs?

Reply:
Hmm... I think I might have made a wrong assumption here - do we let single pass tests fail a PR or not?

Reply:
They do fail PRs, so they should be reported.

Reply:
Done